packages/styleguide/src/components/Editor/Forms/Memo/Picker.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import { css } from 'glamor'
+import { MarkerConfig } from '../../../Marker'
+
+const styles = {
+  picker: css({
+    border: '1px solid transparent',
+    borderRadius: '4px',
+    marginRight: 10,
+    width: 32,
+    height: 32,
+    verticalAlign: 'middle',
+    cursor: 'pointer',
+  }),
+}
+
+export const Picker = ({ marker = 'yellow', isSelected, onClick }) => {
```

Comment:
Could easily be TS I think, only special type would be the marker-prop I suppose (already commented on that not being typed in another component)

Reply:
you are, of course, right